frontend/helpers/graphql/mutations/update_answer.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from '@apollo/client'
+
+const UPDATE_ANSWER = gql`
+    mutation UpdateAnswer($id: ID!, $content: String!) {
+        updateAnswer(id: $id, content: $content) {
+            content
+            created_at
+            updated_at
```

Comment:
I don't think you should edit the created_at and updated _at since it's automatically generated.